sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -279,8 +285,8 @@ Connection getConnection() throws Exception {
 
     public Read<T> withStatementPrepator(StatementPreparator statementPreparator) {
```

Comment:
Actually, there's an issue there. It should be `withStatementPreparator()`. Can you fix that as well ?